doc-gen-service/src/v1/routes/doc-gen-route.ts

```diff
@@ -13,6 +13,7 @@ docGenRoute.post('', async (req: Request, res: Response) => {
   const reportData = req.body;
   const reportTypeQs = req.query.reportType;
   try {
+    console.log(reportData);
```

Comment:
use logger than console.log

Reply:
oops, the console.log statement should be there at all.  I removed it.